src/sentry/workflow_engine/migrations/0012_action_updates.py

```diff
@@ -22,7 +22,7 @@ class Migration(CheckedMigration):
     is_post_deployment = False
```

Comment:
should this be marked as `is_post_deployment = True`?

It looks to me that this migration has already been applied: https://redash.getsentry.net/queries/7743/source, and since we changed the file name, i am not sure if django will know that this needs to be skipped

Reply:
Oops, I totally messed this up and renamed the wrong file